app/models/account_moderation_note.rb

```diff
@@ -18,7 +18,7 @@ class AccountModerationNote < ApplicationRecord
   belongs_to :account
   belongs_to :target_account, class_name: 'Account'
 
-  scope :latest, -> { reorder('created_at DESC') }
+  scope :latest, -> { reorder(id: :asc) }
```

Comment:
We use the ID column for the sorting, since there's an index on it (the primary key)

Reply:
The name was always a little awkward.. I'm open to changing the name, I just don't have a better name for it.

Reply:
There's also not pagination or a limit here, so I really don't know what an appropriate name should be.

Perhaps this can just be a default scope instead of a named scope?

Reply:
`latest` to denote this order would definitely be *more* confusing as we use `id: :desc`/`created_at: :desc` for `latest` everywhere else.

I'm not opposed to just explicitly using `order(id: :desc)` on the call sites, it's clearer than `oldest` or anything else we could come up with. Failing that, something like `oldest_first` seems clear enough to me.